response/return.php

```diff
@@ -0,0 +1,19 @@
+<?php
+session_start();
+require_once('../config.php');
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
+    $conn = new mysqli(SERVER_NAME, USER_NAME, PASSWORD, DATABASE_NAME);
+    if ($conn->connect_error) {
+        die( "Connection failed: " . $conn->connect_error );
+    }
+    $email =$_POST['email'];
+    $sql = "SELECT * from users where email = '$email'";
+    $result = $conn->query($sql);
+    if ($result->num_rows == 0) {
+        $error->email = "";
+        echo json_encode($error);
```

Comment:
Please follow DRY (Don't repeat yourself) rule in your code. Line 14 and 17 is same. It could have been moved after the if-else block.